packages/squiggle-lang/__tests__/TS/Symbolic_test.ts

```diff
@@ -0,0 +1,35 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Symbolic mean", () => {
+  let triangularInputError = {
```

Comment:
Yay! We should look into doing more tests around errors

Reply:
Gonna resolve this, possibly start a new thread after I push up the revamped version (there's a try-catch in the test, which may be smelly). Ozzie is skeptical of the value even when you consider testing for error trails.